introduction.md

```diff
@@ -0,0 +1 @@
+'Hello! I am Gaurav, by profession developer. I am here to learn a lot about git hub.' 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.